http_check/tests/test_unit.py

```diff
@@ -22,3 +22,11 @@ def test__init__():
     init_config = {'ca_certs': 'foo'}
     http_check = HTTPCheck('http_check', init_config, [{}])
     assert http_check.ca_certs == 'foo'
+
+
+def test_instances_do_not_share_data():
+    http_check_1 = HTTPCheck('http_check', {'ca_certs': 'foo'}, [{}])
+    http_check_2 = HTTPCheck('http_check', {'ca_certs': 'bar'}, [{}])
+
+    assert http_check_1.HTTP_CONFIG_REMAPPER['ca_certs']['default'] == 'foo'
+    assert http_check_2.HTTP_CONFIG_REMAPPER['ca_certs']['default'] == 'bar'
```

Comment:
Could you also add:

```suggestion
    assert http_check_2.HTTP_CONFIG_REMAPPER['ca_certs']['default'] == 'bar'
    assert HTTPCheck.DEFAULT_HTTP_CONFIG_REMAPPER['ca_certs'] == {'name': 'tls_ca_cert'}
```

To make sure we are not modifying the original class attribute.